scripts/install-git-hooks.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+HOOKS=(pre-commit commit-msg prepare-commit-msg)
+
+# Adds the call to the local hook files to the .git/hooks commands
+# Creates the .git/hooks commands if they don't already exists
+for HOOK in ${HOOKS[@]}
+do
+  HOOK_FILE=.git/hooks/${HOOK}
+  HOOK_CMD="/bin/bash scripts/${HOOK}.sh"
```

Comment:
minor but
`HOOK_CMD="./scripts/${HOOK}.sh"`
should be enough as we already have ` #!/bin/bash` at top of the pre-commit file